docs/4.0. Behaviour.md

```diff
@@ -28,13 +28,17 @@ While an activation is in progress, concurrent GET requests to the `/active` res
 
 If an API implementation receives a new PATCH request to the `/staged` resource while an activation is in progress it SHOULD block the request until the previous activation is complete. Any GET requests to `/staged` during this time MAY also be blocked until the activation is complete.
 
+## Connection Status
+
+The Connection Management API does not take any responsibility for monitoring the status of a connection between a Sender and a Receiver, for example whether a Receiver is currently receiving packets. As such a loss of packets at a connectionless Receiver (e.g. UDP) or a recoverable error on a connection-oriented transport (e.g. TCP) MUST NOT be indicated via changes to the `/active` resource. Where a connection-oriented transport encounters a recoverable error (such as a broken connection or timeout), the underlying Sender or Receiver SHOULD attempt to recover from the error unless otherwise configured not to do so. This may involve the use of exponential backoff up to a defined limit. Where a Sender or Receiver is permanently unable to recover a connection-oriented transport without user interaction, for example due to erroneous configuration which was not identified prior to activation, the implementation MAY choose to reflect this into the `/active` resource by setting the `master_enable` to `false`.
+
```

Comment:
Minor:

"the `master_enable`" --> "`master_enable`"

(also, the term 'false' is not monospaced in other places in this doc)